configuration/cmdline-txt.md

```diff
@@ -20,6 +20,17 @@ There are many kernel command line parameters, some of which are defined by the
  - elevator: specifies the I/O scheduler to use. `elevator=deadline` means the kernel imposes a deadline on all I/O operations to prevent request starvation.
  - quiet: sets the default kernel log level to `KERN_WARNING`, which suppresses all but very serious log messages during boot.
 
+#### Display Entries in FKMS and KMS modes
+
+The firmware already adds a preferred resolution and overscan settings via an entry such as:
+
+```video=HDMI-A-1:1920x1080M@60,margin_left=0,margin_right=0,margin_top=0,margin_bottom=0```
+
+This default entry can be modified by duplicating the line above manually in /boot/cmdline.txt and making required changes to the margin parameters. In addition, it is possible to add rotation and reflect parameters as documented in the standard [Linux framebuffer documentation](https://github.com/raspberrypi/linux/blob/rpi-4.19.y/Documentation/fb/modedb.txt)
```

Comment:
Might be worth changing "line above" to "entry above", as people might think "line" means it has to be on a separate line in `cmdline.txt` ?

Might be worth adding some clarification on if / how this relates to the margin settings in `config.txt`? (are the config.txt entries only used when FKMS isn't being used? Does the firmware copy the margin settings from config.txt into the automatic cmdline.txt entry? etc.) ping @6by9 

Perhaps it's worth explaining that if `video=...` _isn't_ explicitly set in `cmdline.txt`, then the automatic firmware-supplied entry can be read in Raspbian by doing `cat /proc/cmdline`

Is it necessary to explain what the `HDMI-A-1` part means and/or what the available options here are?

Reply:
AIUI, those margin entries come from any config.txt entries for overscan. Have added an extra sentence in first paragraph to explain use of cat /proc/cmdline. Not sure about describing the HDMI-A bit. It is in the linked Linux docs.

Reply:
Displays are HDMI-A-1, HDMI-A-2, DSI-1 (covers DPI as well), and Composite-1.
```HDMI-A``` is because it is an HDMI type A connector. Technically it's a type D on Pi4, and type C on Pi0, but electrically they are identical. Type B isn't, and is the alternative from https://github.com/raspberrypi/linux/blob/rpi-4.19.y/drivers/gpu/drm/drm_connector.c#L71.

Margin properties are generated by the firmware based on the config.txt overscan_ properties.
The firmware ```video=``` can be disabled via ```disable_fw_kms_setup=1```.

Reply:
OK, have added a table and rejigged a few lines.